client/src/js/services/ProjectService.js

```diff
@@ -1,26 +1,24 @@
 angular.module('bhima.services')
 .service('ProjectService', ProjectService);
 
-ProjectService.$inject = [ '$http', 'util' ];
+ProjectService.$inject = [ '$http', '$translate', '$window', 'util' ];
 
-function ProjectService($http, util) {
+function ProjectService($http, $translate, $window, util) {
   var service = this;
 
   service.create = create;
   service.read = read;
   service.update = update;
-  service.delete = del;
+  service.del = del;
```

Comment:
That's a good intention, but the reason we cannot call the function "delete" is because `delete` is a javascript reserved word.  To make the API clear, the API should still be called `service.delete`.